tensorflow/lite/micro/kernels/detection_postprocess.cc

```diff
@@ -350,6 +350,57 @@ void DecreasingPartialArgSort(const float* values, int num_values,
       [&values](const int i, const int j) { return values[i] > values[j]; });
 }
 
+template <typename Compare>
+void insertion_sort(int* start, int* end, Compare compare) {
```

Comment:
let's update all the function names to be CamelCase.

InsertionSort, TopDownMerge ...

Reply:
Will do

Reply:
Done